bin/helm-build

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+# trap the last failed command
+trap 'printf "Error on exit:\n  Exit code: $?\n  Failed command: \"$BASH_COMMAND\"\n"' ERR
+
+bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+rootdir="$( cd $bindir/.. && pwd )"
+
+$bindir/helm lint $rootdir/charts/partials
+
+$bindir/helm dep up $rootdir/charts/linkerd2
+$bindir/helm dep up $rootdir/charts/patch
+$bindir/helm lint --set Identity.TrustAnchorsPEM="fake-trust" --set Identity.Issuer.TLS.CrtPEM="fake-cert" --set Identity.Issuer.TLS.KeyPEM="fake-key" --set Identity.Issuer.CrtExpiry="fake-expiry-date" $rootdir/charts/linkerd2
+
+# bump the Linkerd version in the chart
```

Comment:
are there use cases where we'd want to run `helm-build` but not bump the version? for example, if i run `helm-build` on my local checkout of this branch, i get:

```diff
-appVersion: edge-19.8.3
+appVersion: git-bc86453f
...
-LinkerdVersion: &linkerd_version edge-19.8.3
+LinkerdVersion: &linkerd_version git-bc86453f
```

...which i think we don't want? what about decoupling `helm-build` from the version bumps? we could even make version bumps just a documented part of our release process.

Reply:
Yeah, my initial thought for this and `appVersion` in `Chart.yaml` was to make the version bump a manual step, until we publish the chart. I felt like there was no point of putting a lot of effort into this until we get a chance to iron out our chart publication process. This was a last-minute, minimum-effort addition, which I am happy to remove.